docs/apache-airflow/templates-ref.rst

```diff
@@ -26,12 +26,29 @@ The following come for free out of the box with Airflow.
 Additional custom macros can be added globally through :doc:`/plugins`, or at a DAG level through the
 ``DAG.user_defined_macros`` argument.
 
+.. _templates:basics:
+
+Note on Scheduling Basics
+-------------------------
+
+Airflow schedules tasks at the **end** of the interval (see :doc:`/scheduler`).
+
+Meaning that when you do:
+ | start_date: datetime(2018, 1, 1, 8, 0,0)
+ | schedule_interval: '0 8 * * *'
+ 
+The first run will kick in at ``2018-01-02 at 08:00+-`` (depends on resources) - but ds and ds_nodash will 
+contain ``2018-01-01``/``20180101`` as **logical** start date. That is **yesterday** from the point of view
+of the actual execution day.
+
+(Elad Kalif/Peter Mortensen: https://stackoverflow.com/a/65196624)
```

Comment:
If I recall correct there are already at least two sections explaining this, before this one was added. You might want to look them up and reference those instead of adding an entire new section.

Reply:
Yes, there are. That is why I added the reference as first line (see :doc:`/scheduler`). Unfortunately, the explanation there is not that easy to understand, that as a result "ds" contains "yesterday".

As in the scheduler explanation the reference to ds etc. does not belong (IMHO), I would like to see it here - as by using Google & searching for ds etc. you will like land here, first.

But I agree with you that I can reduce the explanation to a short sentence.

Beside that I have also now an example for the use of macros and I will add it.

Reply:
Would it be a good idea if we *move* some of the content here? The main thing I’m wondering is it is difficult to keep multiple documentation sections in sync (for example if we need to change the semantics of those variables a bit—which we actually just did not too long ago), so minimise duplication would be helpful. If the content reads well here for you, would you say some of the content might not flow as well in their existing locations? (This is purely an open question; I have no answers.)